host/class/uac/usb_host_uac/include/usb/uac_host.h

```diff
@@ -0,0 +1,374 @@
+/*
+ * SPDX-FileCopyrightText: 2023-2024 Espressif Systems (Shanghai) CO LTD
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#pragma once
+
+#include <wchar.h>
+#include <stdint.h>
+#include "esp_err.h"
+#include "uac.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @brief USB UAC HOST string descriptor maximal length
+ *
+ * The maximum possible number of characters in an embedded string is device specific.
+ * For USB devices, the maximum string length is 126 wide characters (not including the terminating NULL character).
+ * This is a length, which is available to upper level application during getting information
+ * of UAC Device with 'uac_host_get_device_info' call.
+ *
+ * To decrease memory usage 32 wide characters (64 bytes per every string) is used.
+*/
+#define UAC_STR_DESC_MAX_LENGTH              (32)
+#define FLAG_STREAM_SUSPEND_AFTER_START      (1 << 0)      /*!< suspend stream after start, then call uac_host_device_resume to resume */
```

Comment:
Could we add a better description about where this flag is supposed to be used?

Reply:
ok, added